okon_actions.py

```diff
@@ -0,0 +1,390 @@
+""" Module with implementation of OKON actions using py_trees
+
+"""
+import math
+import sys
+import time
+
+import py_trees
+
+from okon_client import Okon
+
+
+class SetDepth(py_trees.behaviour.Behaviour):
+
+    def __init__(self,
+                 name: str = "set depth",
+                 okon: Okon = None,
+                 depth: float = 0.6,
+                 delta: float = .05):
+        super().__init__(name)
+        self.okon = okon
+        self.depth = depth
+        self.delta = delta
+        self.logger.debug("%s.__init__()" % (self.__class__.__name__))
+
+    def initialise(self):
+        self.logger.debug("%s.initialise()" % (self.__class__.__name__))
+
+    def update(self) -> py_trees.common.Status:
+        self.okon.set_depth(self.depth)
+        new_status = py_trees.common.Status.SUCCESS if self.okon.reachedTargetDepth(
```

Comment:
if you use `py_trees.common.Status.SUCCESS` often maybe consider making something like "python typedef", which could short this long name

Reply:
Same for the other py_trees stuff like `FAILURE`, `READ` and `WRITE`